kong/db/dao/init.lua

```diff
@@ -158,6 +166,7 @@ function _M.new(schema, strategy, errors)
     schema   = schema,
     strategy = strategy,
     errors   = errors,
+    super    = DAO, -- allows custom daos to do self.super.delete(self, ...)
```

Comment:
I see, but this feels like OO terminology. Maybe just call this `dao`.

Reply:
I see what you say but I think calling it `dao` would also be confusing as well ("I'm using the DAO while building my DAO").

May I suggest `base` instead?

Reply:
(we spoke offline and we decided to leave it as `super`)